### ISSUE TITLE
Default port 5000 conflicts with widely used mDNSResponder

### ISSUE THREAD

**Issue by katyo** [HBehrens/puncover#44]
Many linuxes and macos has upnp service on port 5000 (mDNSRespnder from Bonjour / Rendezvous).
By default puncover tries to use same port what leads to conflict:

```
Traceback (most recent call last):
  File "/home/kayo/devel/zephyr/.venv/bin/puncover", line 33, in <module>
    sys.exit(load_entry_point('puncover==0.0.1', 'console_scripts', 'puncover')())
  File "/home/kayo/devel/zephyr/.venv/lib/python3.8/site-packages/puncover/puncover.py", line 65, in main
    app.run(host=args.host, port=args.port)
  File "/home/kayo/devel/zephyr/.venv/lib/python3.8/site-packages/flask/app.py", line 772, in run
    run_simple(host, port, self, **options)
  File "/home/kayo/devel/zephyr/.venv/lib/python3.8/site-packages/werkzeug/serving.py", line 1008, in run_simple
    inner()
  File "/home/kayo/devel/zephyr/.venv/lib/python3.8/site-packages/werkzeug/serving.py", line 948, in inner
    srv = make_server(
  File "/home/kayo/devel/zephyr/.venv/lib/python3.8/site-packages/werkzeug/serving.py", line 795, in make_server
    return BaseWSGIServer(
  File "/home/kayo/devel/zephyr/.venv/lib/python3.8/site-packages/werkzeug/serving.py", line 686, in __init__
    super().__init__(server_address, handler)  # type: ignore
  File "/nix/store/zv167zw7gwr9h0gpkfcikdfqnscrd0q6-python3-3.8.9/lib/python3.8/socketserver.py", line 452, in __init__
    self.server_bind()
  File "/nix/store/zv167zw7gwr9h0gpkfcikdfqnscrd0q6-python3-3.8.9/lib/python3.8/http/server.py", line 138, in server_bind
    socketserver.TCPServer.server_bind(self)
  File "/nix/store/zv167zw7gwr9h0gpkfcikdfqnscrd0q6-python3-3.8.9/lib/python3.8/socketserver.py", line 466, in server_bind
    self.socket.bind(self.server_address)
OSError: [Errno 98] Address already in use
```

**Comment by HBehrens** [comment]
There is a command line argument `--port` that allows you to specify the port that defaults to 5000. I'd be happy to merge a pull request that prints a suggestion for that one whenever this specific errors occurs. That way, the feature is more discoverable for other users in the future.

https://github.com/HBehrens/puncover/blob/706dcc5e4a7177817b8cf0830b6894ba2d4320e3/puncover/puncover.py

**Comment by noahp** [comment]
This is further addressed in #71 and released as version 0.3.0 to pypi.
